app/uk/gov/hmrc/gform/gformbackend/GformConnector.scala

```diff
@@ -102,7 +103,7 @@ class GformConnector(ws: WSHttp, baseUrl: String) {
   def validateSection(formId: FormId, sectionNumber: SectionNumber)(
     implicit hc: HeaderCarrier,
     ec: ExecutionContext): Future[String] =
-    ws.GET[String](s"$baseUrl/forms/${formId.value}/validate-section/${sectionNumber.value}")
+    ws.GET[String](s"$baseUrl/forms/${formId.value}//${sectionNumber.value}")
```

Comment:
This was a mistake edit?

That `validateSection` is never called, so it can be removed completely, your call.